dbt/models/iasworld/schema/iasworld.comdat.yml

```diff
@@ -74,7 +74,8 @@ sources:
                       AND deactivat IS NULL
                       AND calc_meth IS NOT NULL
                   meta:
-                    description: calc_meth should be 'E' if not null
+                    description: >
+                      calc_meth (Calculation Method) should be 'E' if not null
```

Comment:
Note that I've only updated the descriptions for these new calculated fields in `comdat` and `oby`, since they are identical to the ones in `dweldat` that are also fixed as of this PR. If you think we should do a full pass on _all_ fields in `comdat` and `oby`, and not just these new calculated fields, I'm happy to do so as well; this PR is just following the instructions of https://github.com/ccao-data/data-architecture/issues/354 somewhat narrowly.